helper/helper_test.go

```diff
@@ -98,4 +265,235 @@ var _ = Describe("helper tests", func() {
 			Expect(isSubset).To(Equal(false))
 		})
 	})
+
+	Describe("doRestoreAgent Mocked unit tests", func() {
+		BeforeEach(func() {
+			// Setup mocked tests environment
+			*singleDataFile = false
+			*content = 1
+			*oidFile = "testoid.dat"
+			*isResizeRestore = true
+			*origSize = 5
+			*destSize = 3
+			*pipeFile = "mock"
+			*onErrorContinue = true
+		})
+		It("successfully restores using a single data file when inputs are valid and no errors occur", func() {
+			*singleDataFile = true
+
+			oidBatch := []oidWithBatch{{oid: 1, batch: 1}}
+			steps := []helperTestStep{
+				{},
+				{getRestorePipeWriterArgExpect: "mock_1_1", getRestorePipeWriterResult: true, checkSkipFileArgTableOid: 1, checkSkipFileResult: false},
+			}
+
+			mockHelper := newHelperTest(oidBatch, steps)
+
+			// Prepare and write the toc file
+			testDir := "" //"/tmp/helper_test/20180101/20180101010101/"
+			*tocFile = fmt.Sprintf("%stest_toc.yaml", testDir)
+			writeTestTOC(*tocFile)
```

Comment:
Why do we need a toc file in this test, but not in the next one?

Reply:
Perhaps because ToC file used for single data file and not used for multiple data file restore.